hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/datanodes/datanodes.tsx

```diff
@@ -420,6 +423,25 @@ export class Datanodes extends React.Component<Record<string, object>, IDatanode
       showDataFetchError(error.toString());
     });
   };
+  
+  removeDatanode= async (selectedRowKeys: any) => {
```

Comment:
Just a nit, can we add a space before the `=`.

Reply:
@devabhishekpal  completed changes in latest commit.